spec/services/copy_course_from_production_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CopyCourseFromProduction do
+  let(:url_base) { 'https://dashboard.wikiedu.org/courses/' }
+  let(:existent_prod_course_slug) do
+    'University_of_South_Carolina/Invertebrate_Zoology_(Spring_2022)'
+  end
+
+  context 'with courses in production' do
+    let(:subject) do
+      described_class.new(url_base + existent_prod_course_slug)
```

Comment:
Since these tests make external web requests from the server, you need to use VCR to allow those requests to go through in the test environment.

Reply:
@ragesoss I just realize about this comment, in the morning I added de VCR.use_cassette block to each test case, do you think is ok there?

Reply:
The way you did it should be fine.